presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/BasicConnectorSession.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+import com.facebook.presto.common.function.SqlFunctionProperties;
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.function.SqlFunctionId;
+import com.facebook.presto.spi.function.SqlInvokedFunction;
+import com.facebook.presto.spi.security.ConnectorIdentity;
+
+import java.util.Locale;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * This is a basic connector session. Should only be used to create HdfsContext, in the absence of FullConnectorSession.
+ */
+
+public class BasicConnectorSession
```

Comment:
Maybe call it `EmptyConnectorSession` and only make it a glue thing. Just like the one you did for raptor

Reply:
I need it 2 places, 1 for FileHiveMetaStore and 2nd one for Glue